### PR TITLE
Finished FlashCards

### PR DESCRIPTION
**Pull request by ManuelE-Osorio**
Finished the project. There are improvement points as shown in the Readme file, but I think its good enough to continue.

### CODE REVIEW

**Comment by ManuelE-Osorio** [comment]
Seems codacy just got stuck, because it hasn't started the analysis  .

**Comment by ManuelE-Osorio** [comment]
Seems codacy just got stuck.

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@ManuelE-Osorio My apologies for the delay. 🥺 I'm getting this error when deleting a stack:

image

File: FlashCards.ManuelE-Osorio/Flashcards/Controllers/DataController.cs [hunk 1]
```diff
@@ -238,7 +238,7 @@ public void SelectOrDeleteStackController(string selectionString)
                 SelectedStack = currentStacks.Find(stacks => stacks.StackName == stackName);
                 break;
             case("delete"):
-                DBController.DeleteStack(SelectedStack);
+                DBController.DeleteStack(new Stacks(stackName));
                 if (SelectedStack?.StackName == stackName)
```
Comment (ManuelE-Osorio) [inline FlashCards.ManuelE-Osorio/Flashcards/Controllers/DataController.cs:242]: Fixed bug in line 241

**Comment by ManuelE-Osorio** [comment]
Fixed the bug. I was passing the Selected Stack and not the user input to the delete method. Also Codacy seems to be working now haha

**Comment by TheCSharpAcademy** [comment]
@ManuelE-Osorio Great project, well done! 🤓 I think the only thing I could have added is more clarification regards to selecting a stack. You could keep an UI element so the users now which stacks are selected at any point in time (if any).

**Pull request merged**